llvm/lib/Target/AIE/AIECombinerHelper.cpp

```diff
@@ -1344,10 +1344,24 @@ bool llvm::matchUpdToConcat(MachineInstr &MI, MachineRegisterInfo &MRI,
   return true;
 }
 
+MachineInstr &findClosestToUseInsertPoint(MachineInstr &MI,
+                                          MachineRegisterInfo &MRI) {
+
+  for (auto &User : MRI.use_instructions(MI.getOperand(0).getReg())) {
+    if (User.isPHI())
+      continue;
+    if (User.getParent() == MI.getParent() && canDelayMemOp(MI, User, MRI))
```

Comment:
Super-nit: I think we should really rename `canDelayMemOp` into just `canDelayOp`